app/characterization_services/mediainfo_characterization_service.rb

```diff
@@ -78,7 +73,11 @@ def valid?
     # Determine if the media type for the FileSet is supported
     # @return [TrueClass, FalseClass]
     def supported_format?
-      !(@file_set.mime_type & self.class.supported_formats).empty? || preservation_file&.original_filename&.first&.downcase&.include?(".wav")
+      !(@file_set.mime_type & self.class.supported_formats).empty? || extension&.include?(".wav") || extension&.include?(".mp4")
```

Comment:
Can we use `.present?` instead of not empty? It seems like using empty would give us errors in the case of certain types of inputs but the set operation should always return a reasonable value.